themes-default/slim/src/components/subtitle-search.vue

```diff
@@ -0,0 +1,258 @@
+<template>
+<!-- template for the subtitle-search component -->
+<tr class='subtitle-search-wrapper'>
```

Comment:
I think the root element should have an indent (makes it easier to read the template)